### PR TITLE
work in progress -tweaks to rich-licence

### PR DESCRIPTION
**Pull request by CountCulture**

### CODE REVIEW

**Comment by inglesp** [comment]
@CountCulture -- this is ready for review, I think.

File: schemas/includes/person.json [hunk 1]
```diff
@@ -15,6 +15,10 @@
       "type": "string",
       "description": "Name of the jurisdiction in which the entity is based"
     },
+    "company": {
```
Comment (CountCulture) [inline schemas/includes/person.json]: @inglesp Did we discuss having relationship_with_company as a field? Thought we did, but not sure we got further. I'm thinking of something like an enum: ['employee', 'director', 'shareholder']

File: spec/sample-data/valid/rich-licence-02.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        "district_office_code": "08",
+        "duplicate_counts": "",
+        "fee_codes": "P40",
+        "geo_code": "3309",
+        "master_indicator": "Y",
+        "premise_census_tract": "0455.02",
+        "premise_county": "RIVERSIDE",
+        "term_in_months": "12"
+    },
+    "jurisdiction_of_licence" => "us_ca",
```
Comment (CountCulture) [inline spec/sample-data/valid/rich-licence-02.json]: typo?

Comment (inglesp) [inline spec/sample-data/valid/rich-licence-02.json]: Give me a clue?

File: schemas/includes/person.json (hunk 1 above)
Comment (inglesp) [inline schemas/includes/person.json]: We certainly discussed it, but I don't think we made any decisions.  Are you suggesting that `relationship_with_company` would be a property inside a person, or inside a person's company?  I think the former makes more sense.

File: spec/sample-data/valid/rich-licence-02.json (hunk 2 above)
Comment (CountCulture) [inline spec/sample-data/valid/rich-licence-02.json]: line 33: "jurisdiction_of_licence" => "us_ca"

don't think that's json, no?

Comment (inglesp) [inline spec/sample-data/valid/rich-licence-02.json]: You're right.  That's odd.

File: schemas/includes/person.json (hunk 1 above)
Comment (CountCulture) [inline schemas/includes/person.json]: I was thinking inside a person, in the same way we have entity_type and entity.

**Pull request merged**